hadoop-ozone/recon/pom.xml

```diff
@@ -103,7 +103,7 @@
               <goal>install-node-and-npm</goal>
             </goals>
             <configuration>
-              <nodeVersion>v16.2.0</nodeVersion>
+              <nodeVersion>v16.14.2</nodeVersion>
```

Comment:
we want to stick to v16 release line only? Guess [17.7.2](https://www.npmjs.com/package/node/v/17.7.2) is also there. Just wanted to confirm

Reply:
Yup. LTS v16 is the way to go, supported till 2024-04-30. While v17 EoL is 2022-06-01. https://nodejs.org/en/about/releases/

Reply:
Makes sense, thnx